<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleQuery</class>
 <widget class="QDialog" name="RuleQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看当前任务</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>476</x>
         <y>0</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonFilter">
          <property name="text">
           <string>过滤</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonQuit">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <column>
       <property name="text">
        <string>任务名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>任务描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务中间结果</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务责任人</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务起始日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务结束日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务实际完成日期</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>规则关键字过滤：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditKeywords">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
